### DISCUSSION TITLE
SCRUM: Reunión 23-08-2023

### DISCUSSION THREAD

**Discussion by RamiroV13** [adempiere/adempiere#4143] [General]
En base a la reunión del día miércoles 23 cuyo temario fue:

1.- cerrar tema ciudadanos
2.- dejar operativo el server para comenzar la carga de Socios de negocio
3.- fijar fecha "Bug fix day"

Los acuerdos que se llegaron fueron:

-Se confirma "bug fix day" los días viernes enfocado para técnicos partiendo el 1 de septiembre
-Víctor trabajará en subir un ambiente 394 para cargar los socio de negocio.
-OFB cargará los socio de negocio/usuarios de los ciudadanos (citizens) y así poblar la BBDD.
-Víctor trabajara con Horacio para el servidor de correo

**Comment by marcalwestf** [comment]
favor confirmar acta.
